src/LinuxTracing/SampleProgram.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_format.h>
+#include <string.h>
+
+#include <atomic>
+#include <chrono>
+#include <iostream>
+#include <mutex>
+#include <thread>
+
+static std::atomic<bool> KEEP_GOING;
+static std::atomic<int> num;
+
+std::mutex mtx;
+
+constexpr int kNumOfThreads = 4;
+
+void DoWork(int thread_num) {
+  std::string thread_name = absl::StrFormat("Thread %d", thread_num);
+  pthread_setname_np(pthread_self(), thread_name.c_str());
+  while (true) {
+    if (!KEEP_GOING.load()) {
+      break;
+    }
+    int temp = num;
```

Comment:
Since you've been using `load` for the other atomic I would also use `load` here.